### PR TITLE
Fix sample:demo rake task

### PR DESCRIPTION
**Pull request by masonforest**
* Refactor compute_current_status!
* Refactor monthly_spend

### CODE REVIEW

**Comment by chriskacerguis** [comment]
:+1:

**Comment by calebthompson** [comment]
:+1:

File: app/models/project.rb [hunk 1]
```diff
@@ -66,10 +66,11 @@ def order_history
   end
 
   def compute_current_status!
-    self.status = latest_alerts.reduce('unknown') do |m, r|
-      STATES[r.status.downcase] > STATES[m] ? r : m
-    end.to_sym
-    save!
+    if latest_alerts.any?
+      update(status: highest_priority_latest_alert.status)
```
Comment (mafernando) [inline app/models/project.rb]: @masonforest - i'm seeing 
image

when I try to run `rake sample:demo`

I think this line should be `update(status: highest_priority_latest_alert.status.downcase)`

Comment (mafernando) [inline app/models/project.rb]: NVM - scooped it up in PR https://github.com/projectjellyfish/api/pull/577

Comment (masonforest) [inline app/models/project.rb]: ah, good catch. Thanks @mafernando!

**Pull request merged**